base_layer/wallet/src/util/emoji.rs

```diff
@@ -122,7 +122,7 @@ impl EmojiId {
         EmojiId::byte_vec(&self.0).unwrap()
     }
 
-    fn from_bytes(bytes: &[u8]) -> Self {
+    pub fn from_bytes(bytes: &[u8]) -> Self {
```

Comment:
This isn't meant to be public because it doesn't do any error checking. You'll notice that it's only called from `from_pubkey` which is guaranteed to be a valid byte string (i.e. 32 bytes long).

Reply:
Re-based on PR#1469